framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -167,6 +168,7 @@ qx.Mixin.define("qx.ui.core.MExecutable",
             // check also for themed values [BUG #5906]
             if (selfPropertyValue == null) {
               // update the appearance to make sure every themed property is up to date
+              this.$$resyncNeeded = true;
```

Comment:
I tried various ways. That was the only way to achieve it without changing too much. As it is internal like the other $$ variables, I don't see a real problem there.